gateway/ln-gateway/src/events.rs

```diff
@@ -0,0 +1,51 @@
+use fedimint_core::core::ModuleKind;
+use fedimint_core::Amount;
+use fedimint_eventlog::{Event, EventKind};
+use fedimint_lnv2_common::contracts::{Commitment, OutgoingContract};
+use serde::{Deserialize, Serialize};
+
+use crate::sha256::Hash;
+
+#[derive(Serialize, Deserialize)]
+pub struct OutgoingPaymentSuccess {
```

Comment:
https://github.com/fedimint/fedimint/blob/master/modules/fedimint-mint-client/src/event.rs#L22

Let's keep the naming to "past verb" `OutgoingPayementSucceded`?